.github/workflows/i18n-push.yml

```
@@ -10,60 +10,8 @@ on: workflow_dispatch # manually trigger
 
 jobs:
   i18n-extract:
-    runs-on: ubuntu-20.04
-    env:
-      PYTHON-VERSION: 3.9
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      # setup python
-      - name: Set up Python ${{ env.PYTHON-VERSION }}
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ env.PYTHON-VERSION }}
-
-      # install dependencies
-      - name: Install dependencies
-        run: |
-          pip install -e ".[all]"  
-
-      # install transifex client
-      - name: transifex-client
-        run: |
-          pip install transifex-client
-
-      # store token
-      - name: store token
-        run: |
-          touch ~/.transifexrc
-          echo [https://www.transifex.com] >> ~/.transifexrc
-          echo api_hostname=https://api.transifex.com >> ~/.transifexrc
-          echo hostname=https://www.transifex.com >> ~/.transifexrc
-          echo password=${{ secrets.TRANSIFEX_TOKEN }} >> ~/.transifexrc
-          echo username=api  >> ~/.transifexrc
-
-      # extract
-      - name: extract_messages
-        run: |
-          python setup.py extract_messages
-
-      # push source
-      - name: push_messages
-        run: |
-          tx push -s
-
-      # remove transifex credential file
-      - name: remove transifexrc file
-        uses: JesseTG/rm@v1.0.2
-        with:
-          path: ~/.transifexrc
-
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v3
-        with:
-          commit-message: i18n:push translations
-          title: i18n:push translations
-          body: i18n:push translations
-          branch: 101-translations-push  # name of branch which PR is created from
-          delete-branch: true  # delete branch once merged,closed
+    uses: inveniosoftware/invenio-i18n/.github/workflows/reusable/i18n-push.yml
+    with:
+      extract-backend: true
+    secrets:
+      TRANSIFEX_TOKEN: ${{ secrets.TRANSIFEX_TOKEN }}
```

```suggestion
    secrets:
      secrets: inherit
```
Read the [docs](https://github.blog/changelog/2022-05-03-github-actions-simplify-using-secrets-with-reusable-workflows/)

---

Thanks. I misunderstood the `inherit` concept first time reading the docs.